content/blog/2024-01-01-flash-intro.html

```diff
@@ -9,8 +9,8 @@
   - nolan
   - richa
   - tina
-heroImage: flash_hero.jpg
-heroImageThumb: flash_thumb.png
+heroImage: blog-lg-flash.jpeg
+heroImageThumb: blog-thumb-flash.jpeg
```

Comment:
You should make these changes in the associated `.Rmd` file as well, and also delete the `flash_hero.jpg` and `flash_thumb.png` files that are no longer being referenced.